### ISSUE TITLE
BuildLandingPad

### ISSUE THREAD

**Issue by onionhammer** [mjsabby/LLVMSharp#43]
https://github.com/mjsabby/LLVMSharp/blob/master/IRBuilder.cs#L135

When I run 
```sh build.sh /usr/local/lib/llvm-3.7/lib/libLLVM-3.7.1.dylib /usr/local/lib/llvm-3.7/include```

I get

```
IRBuilder.cs(135,25): error CS1501: No overload for method `BuildLandingPad' takes `4' arguments
Generated.cs(2069,43): (Location of the symbol related to previous error)
```

Should this call have 5 args in LLVM 3.7?

**Comment by TChatzigiannakis** [comment]
Yes, the signature seems to have changed across versions.

`IRBuilder` and other wrapper classes are still half-baked in the master branch, unfortunately. If they're causing you trouble, I'd say you can safely exclude them from your build until we merge the finished versions to the master branch. In the meantime, you can use `LLVMBuilderRef` instead.

**Comment by mjsabby** [comment]
@onionhammer yeah they added new stuff in this, the repo is now updated. @TChatzigiannakis  is right, we'll be introducing a new OOAPI in the next release, and also make it run on Linux and MacOSX!

**Comment by tannergooding** [comment]
This is removed/replaced API, not actionable

**Comment by tannergooding** [comment]
I'd note for this one in particualr though that `IRBuilder.BuildLandingPad` does exist now and is cleaned up.

We had a big rewrite of everything between the much older OO API surface and what exists today, with a lot of cleanup and fixes.
